python/pyspark/mllib/recommendation.py

```diff
@@ -85,6 +85,12 @@ class MatrixFactorizationModel(JavaModelWrapper, JavaSaveable, JavaLoader):
     >>> len(latents) == 4
     True
 
+    >>> products_for_users = model.recommendProductsForUsers(1).collect()
+    ...
```

Comment:
The output is deterministic. So we should do the following:

~~~python
 >>> model.recommendProductsForUsers(1).collect()
// put expected output here
~~~